hoodie-client/src/main/java/com/uber/hoodie/table/HoodieCopyOnWriteTable.java

```diff
@@ -159,11 +160,22 @@ public boolean isWorkloadProfileNeeded() {
     return true;
   }
 
+  @Override
+  public HoodieCompactionWorkload scheduleCompaction(JavaSparkContext jsc, String commitTime) {
+    throw new HoodieNotSupportedException("Compaction is not supported from a CopyOnWrite table");
+  }
+
   @Override
   public JavaRDD<WriteStatus> compact(JavaSparkContext jsc, String commitTime) {
     throw new HoodieNotSupportedException("Compaction is not supported from a CopyOnWrite table");
   }
 
+  @Override
+  public JavaRDD<WriteStatus> compact(JavaSparkContext jsc, String compactionInstantTime,
```

Comment:
merge this with above ?

Reply:
Yes, the above method is getting removed in subsequent PR